feat: Sellers directory에 결제화면 구현시도

syeobnn opened the pull request:
index.jsx에 나와있는 안내에 따라 구현해야 하는 페이지 경로를 참고
partners에서 판매자 페이지가 구현되어 sellers를 결제화면으로 설정
안내된 경로를 참고해 디렉토리를 만들고 구현을 시도했지만 화면에 나오지 않음

syeobnn committed:
feat: Sellers directory에 결제화면 구현시도

index.jsx에 나와있는 안내에 따라 구현해야 하는 페이지 경로를 참고
partners에서 판매자 페이지가 구현되어 sellers를 결제화면으로 설정
안내된 경로를 참고해 디렉토리를 만들고 구현을 시도했지만 화면에 나오지 않음

return0927 committed:
Merge branch 'dev' into impl/logged_in

return0927 committed:
Merge branch 'dev' into impl/logged_in

return0927 commented on src/pages/sellers/orderpage.jsx:
```diff
@@ -0,0 +1,91 @@
+
+const OrderPage = () => {
+  return (
+    <div className="order-page">
+      <header className="header">
+        <div className="logo">로고</div>
+        <nav className="nav">
+          <a href="/">홈</a>
+          <a href="/mypage">마이페이지</a>
+          <input type="text" placeholder="Search in site" />
+        </nav>
+      </header>
```
헤더는 `src/App.jsx` 에서 자동으로 붙여줍니다.

return0927 reviewed (changes requested):
결제하는 상품은 products 이기 때문에 이 페이지는 `src/pages/products/checkout/$id.jsx` 가 되어야할 것 같습니다.

input element 의 placeholder 나 너비 등 css 조금 신경써주세요!

syeobnn committed:
feat: Coffee web에서 product로 경로 수정 및 디자인 수정

review받은 내용을 적용해 파일 경로 설정
구매 페이지 디자인 수정
이용약관 항목 추가

syeobnn committed:
Merge branch 'impl/logged_in' of https://github.com/return0927/coffeebean-front into impl/logged_in

syeobnn committed:
Merge branch 'dev' of https://github.com/return0927/coffeebean-front into impl/logged_in

syeobnn committed:
feat: Request신청 화면 구현

홈 화면에서 request를 눌렀을 때 request신청 화면이 나오도록 구성
물론 처음에 나와야하는건 request리스트 페이지이므로 나중에 수정 요망
그렇기때문에 나중에 디렉토리도 수정이 필요할 것이라 생각

syeobnn committed:
Delete src/pages/custom/customorder.css

삭제한줄알았는데 잘못 들어감

syeobnn committed:
Delete src/pages/custom/customorder.jsx

얘도 잘못들어감
